### PR TITLE
fix: token list load issue WEB-2083

### PR DESCRIPTION
**Pull request by JFrankfurt**
https://uniswaplabs.atlassian.net/browse/WEB-2877?atlOrigin=eyJpIjoiYjFkNDYxNWIyMWJhNDAwMzkxMWFhOWMzNTY5MGIyZWEiLCJwIjoiaiJ9

### CODE REVIEW

File: src/utils/nativeTokens.ts [hunk 1]
```diff
@@ -4,6 +4,9 @@ import { CHAIN_NAME_TO_CHAIN_ID } from 'graphql/data/util'
 
 export function getNativeTokenDBAddress(chain: Chain): string | undefined {
   const pageChainId = CHAIN_NAME_TO_CHAIN_ID[chain]
+  if (pageChainId === undefined) {
+    return undefined
+  }
```
Comment (just-toby) [inline src/utils/nativeTokens.ts:9]: why is this not caught by the default case below? is this related to the token lists changes?

**Review by just-toby — APPROVED** [review]
list logic changes LGTM

File: src/utils/nativeTokens.ts (hunk 1 above)
Comment (JFrankfurt) [reply src/utils/nativeTokens.ts:9]: this actually is, but I was working w/ the tsconfig suggestion @tinaszheng made in the linked slack thread and it flagged this case too, so why not?
